File: techila/examples/R/Tutorial/4_multiplefunctions/multi_function_dist.r
# Copyright 2010-2013 Techila Technologies Ltd.

# This R-script will be sourced at the preliminary stages of a
# computational Job. The value of input argument in the Local Control
# Code will determine, which function will be called on the Worker.

function1 <- function() {
  # When called in the computational Job, the function return the value 2.
  result <- 1 + 1
}

function2 <- function() {
  # When called in the computational Job, the function return the value 100.
  result <- 10 * 10
}

File: techila/examples/python/Tutorial/4_multiplefunctions/multi_function_dist.py
# Copyright 2012-2013 Techila Technologies Ltd.

# This Python-script will be evaluted at the preliminary stages of a
# computational Job. When evaluated, two functions will be defined.
# Either one of these functions can be then used as an entry point 
# by defining the applicable function name  as the value of
# the funcname parameter in the Local Control Code.

def function1():
    # When called, this function will return the value 2.
    return(1 + 1)

def function2():
    # When called, this function will return the value 100.
    return(10 * 10)
